src/_includes/components/sub-banner.njk

```diff
@@ -0,0 +1,21 @@
+<ul>
+    <li>
+    <img src="" alt="octogon shape here"/>
```

Comment:
@will0101 This might not have be 100% clear in the mock ups, but the first item on this navbar is a link back to the VSP landing page, not just a decorative image.

Reply:
Fixed, plus I just noticed I misspelled octagon 💀.